versions.yaml

```diff
@@ -164,7 +164,7 @@ assets:
     url: "https://cdn.kernel.org/pub/linux/kernel/v4.x/"
     uscan-url: >-
       https://mirrors.edge.kernel.org/pub/linux/kernel/v4.x/linux-(4\.19\..+)\.tar\.gz
-    version: "v4.19.28"
+    version: "v4.19.52"
```

Comment:
upstream is at .53 already - just to note. .52 has the CVE fix in it though, so we can roll with this.